.github/workflows/upload-android-test-specs.yml

```diff
@@ -4,13 +4,13 @@ on:
   pull_request:
     paths:
       - .github/workflows/upload-android-test-specs.yml
-      - examples/demo-apps/android/LlamaDemo/android-llm-device-farm-test-spec.yml
+      - extension/android/benchmark/android-llm-device-farm-test-spec.yml
```

Comment:
should we rename it as we are going to use it for both llm and non-genai mdoels

Reply:
Good point! Honestly there are more stuff to modify in this yml, so I will do it later together.